code/modules/mob/living/basic/alien/queen.dm

```diff
@@ -37,4 +37,5 @@
 	)
 	mob_size = MOB_SIZE_LARGE
 	gold_core_spawnable = NO_SPAWN
-
+	pixel_x = -16
```

Comment:
?

Reply:
should this set base_pixel_x too?

Reply:
Uses the same sprite as the `carbon/alien` one but doesn't have the offset, so it wasn't centered on any particular tile. Now it is.
Also yeah it should.